### DISCUSSION TITLE
Birthdays?

### DISCUSSION THREAD

**Discussion by IamPatrickSturgill** [marcusolsson/obsidian-projects#904] [General]
Hello,
So I'm experimenting with using Projects for my tasks instead of Todoist. One of the things I do on todoist is I have "important dates" project that I have people's birthdays, remembrance days, and anniversaries. Then I use the todoist plugin to bring in to my daily notes. Obviously, I could do this once with projects, but is there a way to do a recurring date or a way to just do the month and day but not the year? Thanks!

**Comment by HilkopterBob** [comment]
@marcusolsson is the functionality given?
